src/plugins/presentation_util/public/components/input_controls/control_frame/control_frame.tsx

```diff
@@ -29,14 +30,20 @@ export const ControlFrame = ({ twoLine, embeddable }: ControlFrameProps) => {
   const form = (
     <EuiFormControlLayout
       className="controlFrame--formControlLayout"
-      fullWidth={true}
```

Comment:
I believe the plan is for the size of individual controls to be handled by the `control group`, via passing props to the control frame. Likely full width will need to be used to ensure that it takes up the full space given to it whether it is set to 'Fill unused space', or set to a specific pixel width.

With `fullWidth` removed, you can end up in funny situations where the gutter sizes between controls get much bigger than they should, or that the controls leave a lot of blank space on the right side. I think it would be better to leave this enabled for now, and talk about controlling the width of individual controls when we get to the `control group` PR.

<img width="711" alt="Screen Shot 2021-07-30 at 12 34 00 PM" src="https://user-images.githubusercontent.com/14276393/127677106-b41430a1-727c-45f5-a8aa-7b1151244827.png">
<img width="986" alt="Screen Shot 2021-07-30 at 12 33 45 PM" src="https://user-images.githubusercontent.com/14276393/127677110-73cee250-1ad9-4b16-b612-e6535cb9fb6a.png">

Note that these behaviours only show up with a min-width set.

Reply:
Added `fullWidth` back